src/families/tron/cli-transaction.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+
+import invariant from "invariant";
+import { getAccountCurrency } from "../../account";
+import type {
+  Transaction,
+  Account,
+  AccountLike,
+  AccountLikeArray
+} from "../../types";
+
+const options = [
+  {
+    name: "token",
+    alias: "t",
+    type: String,
+    desc: "use an token account children of the account"
+  },
+  {
+    name: "mode",
+    type: String,
+    desc: "mode of transaction: send, freeze, unfreeze"
+  },
+  {
+    name: "duration",
+    type: String,
+    desc: "duration in day"
+  },
+  {
+    name: "resource",
+    type: String,
+    desc: "reward ENERGY or BANDWIDTH"
+  }
+];
+
+function inferAccounts(account: Account, opts: Object): AccountLikeArray {
+  invariant(account.currency.family === "tron", "tron family");
+  if (!opts.token) return [account];
+  return opts.token.map(token => {
+    const subAccounts = account.subAccounts || [];
+    if (token) {
+      const subAccount = subAccounts.find(t => {
+        const currency = getAccountCurrency(t);
+        return (
+          token.toLowerCase() === currency.ticker.toLowerCase() ||
+          token.toLowerCase() === currency.id
+        );
+      });
+      if (!subAccount) {
+        throw new Error(
+          "token account '" +
+            token +
+            "' not found. Available: " +
+            subAccounts.map(t => getAccountCurrency(t).ticker).join(", ")
+        );
+      }
+      return subAccount;
+    }
+  });
+}
+
+function inferTransactions(
+  transactions: Array<{ account: AccountLike, transaction: Transaction }>,
+  opts: Object,
+  { inferAmount }: *
+): Transaction[] {
+  return transactions.flatMap(({ transaction, account }) => {
```

Comment:
With the new update of cli, it's seems that transactions don't contain `flatMap` fonction anymore
we should use `loadash` `flatmap` or see if we can just use a `map`